### PR TITLE
fix: Update dependencies and lint

### PR DESCRIPTION
**Pull request by PeterC89**
Update all dependencies and fix linting problems caused by the updates.

Unit tests fail sporadically with:
```zsh
[Error: ENOENT: no such file or directory, rename 'data/serviceMessages.db~' -> 'data/serviceMessages.db'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'rename',
  path: 'data/serviceMessages.db~',
  dest: 'data/serviceMessages.db'
}
```

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (Julusian) [inline .prettierrc.json:1]: This file shouldnt be needed, we specify the config via the `prettier` property in package.json

File: src/api/coreControl/peripheralDevices.ts [hunk 2]
```diff
@@ -12,11 +12,11 @@ export interface SubDeviceStatePayload {
 	desiredSubDeviceState: SofieSubDeviceState
 }
 
-export function isSofieSubDeviceState(obj: any): obj is SofieSubDeviceState {
+export function isSofieSubDeviceState(obj: SofieSubDeviceState): obj is SofieSubDeviceState {
```
Comment (Julusian) [inline src/api/coreControl/peripheralDevices.ts:15]: What was the reason for changing these?
I thought the purpose of these functions were to safely cast something of any type to `SofieSubDeviceState`, if the input already has to be `SofieSubDeviceState` does that mean having to do the cast at each call site before using this to verify the cast?

**Review by Julusian — APPROVED** [review]
I'm personally not a fan of using the `void` keyword to ignore promises like this, I feel like it makes it too easy to accidentally cause a promise to be unhandled while refactoring. 

Instead I would have made a small function which wraps the promisey handlers, and converts them to a non-promise handler and acts as a fallback for handling the error in some sensible way.

I'm not opposed to leaving it as is, as this (as far as Im aware) is a non-critical system

File: src/api/coreControl/peripheralDevices.ts (hunk 2 above)
Comment (PeterC89) [reply src/api/coreControl/peripheralDevices.ts:15]: There's a linter error with `any`: `Argument 'obj' should be typed with a non-any type.`
I've changed it to `Record<string, any>`

File: .prettierrc.json (hunk 1 above)
Comment (PeterC89) [reply .prettierrc.json:1]: Removed

**Comment by PeterC89** [comment]
> I'm personally not a fan of using the `void` keyword to ignore promises like this, I feel like it makes it too easy to accidentally cause a promise to be unhandled while refactoring.
> 
> Instead I would have made a small function which wraps the promisey handlers, and converts them to a non-promise handler and acts as a fallback for handling the error in some sensible way.
> 
> I'm not opposed to leaving it as is, as this (as far as Im aware) is a non-critical system

@Julusian I've had a go at doing something like that in [5beb5cf](https://github.com/nrkno/sofie-monitor/pull/16/commits/5beb5cf6a32520602efd2aed948da3552e5bc726), let me know what you think?

**Pull request merged**